src/assets/templates/typescript/tslint.json

```diff
@@ -4,6 +4,22 @@
       "tslint-config-airbnb"
   ],
   "jsRules": {},
-  "rules": {},
+  "rules": {
+    "max-line-length": [
+      false
+    ],
+    "triple-equals":[
+      false
+    ],
+    "no-else-after-return":[
```

Comment:
✅ This seems like preference to me, but I run into this often too and am also annoyed, so 👍